install-from-binstall-release.ps1

```diff
@@ -0,0 +1,16 @@
+$tmpdir = $Env:TEMP
+$type = (Get-ComputerInfo).CsSystemType
+if ($type.StartsWith("x64")) {
+	$url = "https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-pc-windows-msvc.zip"
+}
+else {
```

Comment:
It would be great if we could check for aarch64 explicitly and rejects other targets.

Reply:
I've also added the explicit check for aarch64, unfortunately I can't test this since I don't have a Windows-on-Arm device, and pwsh on Linux seems to lack the `Get-ComputerInfo` cmdlet.